data/tlg0062/tlg044/tlg0062.tlg044.perseus-eng5.xml

```diff
@@ -0,0 +1,1787 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">Toxaris; Or, Friendship.</title>
+ <author>Lucian of Samosata</author>
+<editor role="translator">Emily James Smith</editor>
+ <funder>National Endowment for the Humanities</funder>
+ <principal>Gregory Crane</principal>
+ <respStmt>
+  <persName>Gregory Crane</persName>
+<resp>Digital conversion and editing</resp>
+ </respStmt>
+ </titleStmt>
+<publicationStmt>
+<authority>Perseus Digital Library</authority>
+<idno type="filename">tlg0062.tlg044.perseus-eng5.xml</idno>
+<availability>
+ <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+</availability>
+<date type="release">2023-08-01</date>
+<publisher>Harvard College Library</publisher>
```

Comment:
Just noted this as well,  how is Harvard College Library the publisher of these?

Reply:
Got it!